crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -543,6 +607,31 @@ impl RoomInfo {
         self.members_synced = false;
     }
 
+    /// Mark this Room still missing some state information
```

Comment:
```suggestion
    /// Mark this Room still missing some state information.
```

Same in a bunch of other places in this PR.